### ISSUE TITLE
[Feature request] Faster camera movement

### ISSUE THREAD

**Issue by Pankeko** [leoetlino/project-restoration#118]
I'm wondering if you could add another version with faster camera movement or one where when you're rolling or swimming you press L to lock the camera a bit far behind and looking straight ahead.
Had some difficulty at the goron shrine inside the moon, game asks you to do 2 90 turns while rolling with a goron and it's really difficult because of the camera, it turns around and makes you go off course.

**Comment by nr1995** [comment]
Is this related to the **“immediate [L] targeting reorientation”** (issue #44)?
You can read about it at the bottom of the list;
it's the earliest still-open issue,
and the one with the most “likes”,
but it's still rated as low priority.

**Comment by leoetlino** [comment]
No, it's entirely unrelated. This issue is about the camera, #44 is about the player controls (if I've understood it correctly).

**Comment by nr1995** [comment]
> No, it's entirely unrelated. This issue is about the camera, #44 is about the player controls (if I've understood it correctly).

The issue I posted (#44) is not exactly about controls.
It's about camera behaviour when you press [L].
In **_Majora's Mask_** **3D** the game forces you to wait after pressing [L],
as if you don't, the camera will go back to where it was before you pressed [L].
In **_Ocarina of Time_** **3D**, the camera will do what you wanted it to,
regardless of if you wait for it or not.

This issue (#188) is not identical, but similar,
i.e. he's complaining that a lack of **fast/responsive** camera controls
makes tight navigation difficult.

**Comment by leoetlino** [comment]
Oh, I see what you mean now. It still sounds like a separate thing to fix, though.

**Comment by nr1995** [comment]
> Oh, I see what you mean now. It still sounds like a separate thing to fix, though.

Yes I think so.
But #44 is still really important!  And probably really easy to fix!
